### PR TITLE
refs #000: improve credential helpers configuration and debugging

### PR DESCRIPTION
**Pull request by Monska85**

### CODE REVIEW

**Review by andypanix — APPROVED** [review]
LGTM

**Review by grota — APPROVED** [review]

File: templates/.gitlab-ci-template.yml [hunk 1]
```diff
@@ -49,23 +49,25 @@ variables:
...
       fi
-      if command -v docker-compose &> /dev/null
-      then
+      if command -v docker-compose &> /dev/null; then
         docker-compose version
       fi
-      if command -v gcloud &> /dev/null
-      then
-        gcloud auth configure-docker
+      if command -v gcloud &> /dev/null; then
```
Comment (paolomainardi) [inline templates/.gitlab-ci-template.yml:63]: should we print here the gcloud cli version ?

**Review by paolomainardi — COMMENTED** [review]
LGTM

File: templates/.gitlab-ci-template.yml (hunk 1 above)
Comment (Monska85) [reply templates/.gitlab-ci-template.yml:63]: Added

**Pull request merged**